src/posts/bookmarks.js

```diff
@@ -5,7 +5,9 @@ const plugins = require('../plugins');
 
 module.exports = function (Posts) {
     Posts.is_important = async function (pid) {
-        return await Posts.getPostField(pid, 'important');
+        // typeof(res) === 'number' or res === null
```

Comment:
Is there a reason that you commented out this type check? 

Otherwise, looks good to me!